chillTime.py

```diff
@@ -79,60 +73,57 @@ def get_scalar(self, obs_type, record, db_manager):
         # Get the first element of the ValueTuple. This will be in Celsius:
         outTemp_F = outTemp_F_vt[0]
         interval_H = interval_H_vt[0]
-        
+
         if self.algorithm == 'simple':
             # Use the "Simple" algorithm. Results will be in hours. Create a ValueTuple out of it:
             if outTemp_F < 45:
                 chill_time = interval_H
             else:
                 chill_time = 0
-            chill_vt = ValueTuple(chill_time, 'hour', 'group_elapsed')
             log.debug("Found Simple chill hours of %s hours", chill_time)
         elif self.algorithm == 'utah':
             # Use the "Utah" algorithm, weighting chill temps between 32 and 45F
             if outTemp_F <= 34:
                 chill_time = 0
-            elif 34 < outTemp_F and outTemp_F <= 36:
+            elif 34 < outTemp_F <= 36:
```

Comment:
This looks a lot cleaner. I only did it this way because a stranger on the internet said that the "and" syntax uses fewer resources.

Reply:
You shouldn't be worrying about optimizations at that level. In any case, I have no doubt that the Python interpretor implements it as an `and` anyway.